src/components/inspector.js

```diff
@@ -135,13 +135,16 @@ const Inspector = props => {
             value={mediaID}
             render={({ open }) => (
               <Fragment>
-                <Button className="button-link" onClick={open}>
+                <div className="editor-post-featured-image__container">
+                  <Button className="components-button editor-post-featured-image__toggle block-bg" onClick={open}>
                   {!mediaID ? (
                     __("Set background image")
                   ) : (
                     <img src={mediaURL} />
                   )}
                 </Button>
+                <div className="components-drop-zone"></div>
```

Comment:
This div is of no use here

Reply:
Solved: [https://github.com/lubusIN/block-background/pull/29/commits/1bba89856e0a332ef9131c8703889fb92c2670d7](https://github.com/lubusIN/block-background/pull/29/commits/1bba89856e0a332ef9131c8703889fb92c2670d7)